GPU/Common/FragmentShaderGenerator.cpp

```diff
@@ -36,13 +36,27 @@
 
 #define WRITE(p, ...) p.F(__VA_ARGS__)
 
+// This is only used in stereo mode, so okay to declare "tex" as ARRAY_ON_
```

Comment:
This comment is slightly misleading now, since it really is talking about `samplersStereo`, I think.  If we're creating all textures with an array image view anyway, it does kinda feel like using an array on Vulkan always might be simpler.  Not sure if we'd encounter some fun Raspberry Pi bug or something...

Although, that's a problem for 3D textures...

-[Unknown]

Reply:
Removed comment.

Yeah.. didn't want to take that step just yet, and as you say, 3D textures complicate things as they can't be arrays.